MAINTAINERS.md

```diff
@@ -0,0 +1,101 @@
+maintainers that collaborated with you and we also don't want to burden the
+senior maintainers otherwise they'll be reluctant to sponsor anyone. Therefore,
+rather than having a mentorship process where a new maintainer is guided by a
+senior member, we want to see the capabilities of the candidate before being
+accepted.
+
+**TODO:** Detail the meaning of a senior project maintainer with a requirement
+along the lines of more than 6 months experience as a project maintainer.
+
+### 4. Apply for maintainership to contributor leads
+
+The application will be discussed by the lead maintainers and put to vote. If
+the vote result is negative, do not be discouraged! You will be given feedback
+so that you can improve yourself and apply again. We want to empower our
+community and cultivate a trust relationship that's why we carefully evaluate
+abilities and experience of each candidate.
+
+**TBD:** As of now Brian Grant is the wg-contribex lead that approves new maintainers.
+
+## What we expect from our maintainers
+
+First and foremost, be a good person, as cheeky as it sounds, you are now a
+member of the Kubernetes project and thus represent the project and your fellow
+developers. We value our community tremendously and we'd like to keep
+cultivating a friendly and collaborative environment for our
+members/contributors/users. For more,
+see
+[Kubernetes Code of Conduct](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/community-expectations.md).
+
+Good technical judgement, and knowledge over one or more components of
+Kubernetes, it might be a project under the Kubernetes or it might be a
+package/component in the main repository such as the Kubernetes documentation.
+We determine the maintainer applicant's judgement via the sponsorship
+requirement. For example, a senior maintainer might decide to sponsor an applicant
+after they witnessed the applicants good technical judgement during a design
+discussion, a feature implementation or by simply witnessing the way they
+interact with other contributors.
+
+## Inactive Maintainers
+
+Maintainers deemed inactive after being inactive for 30 days in the project or
+maintainers that decide to step down will be retired. Retired maintainers might
+decide to come back to the project at a later point. This means that if an
+retired maintainer wants to return, he will be able to regain his access once he
+becomes an active contributor again.
+
+**Motivation:** Since maintainers have already proved themselves by becoming a
+maintainer in the first place, we only require that the maintainer familiarizes
+himself with the project again by becoming an active contributor again.
+Kubernetes as a project is evolving very fast and it requires some time to get
+used to the project changes whether it is around contributor tooling, project
+structure or the codebase itself.
+
+**TBD**: The inactivity period (30 days) is subject to change.
```

Comment:
IMO it should be the same time period we use to determine contributor activity.  This is called out as 2 months in the first bullet point